include/xtensor/xchunked_view.hpp

```diff
@@ -124,6 +142,30 @@ namespace xt
         return *this;
     }
 
+    template <class E>
+    template <class OE>
+    typename std::enable_if<has_chunks<OE>::value, xchunked_view<E>&>::type xchunked_view<E>::operator=(const OE& e)
+    {
+        for (auto it1 = chunk_begin(), it2 = e.chunks().begin(); it1 != chunk_end(); it1++, it2++)
```

Comment:
Since `xchunked_iterator` is a bit heavy to instantiate:

- ++it1, ++it2 to avoid copying iterators.
- chunk_end should be invoked before the loop and stored in a temporary to avoid building it in each iteration.